UPGRADING.md

```diff
@@ -0,0 +1,22 @@
+# Major version upgrades
+
+## 0.x to 1.x
+
+### Deprecations
+
+* FnFormat => FnSub
```

Comment:
When testing with a real world app, I needed to monkey patch FnFormat back in because it is not easy to replace some of the things it did with Fn::Sub - particularly around splitting newlines.